core/src/main/java/io/kestra/core/expression/PebbleFilter.java

```diff
@@ -0,0 +1,23 @@
+package io.kestra.core.expression;
+
+import io.kestra.core.models.annotations.Plugin;
+import io.pebbletemplates.pebble.extension.Filter;
+
+/**
+ * Class for defining a Pebble filter.
+ */
+@Plugin
+public abstract class PebbleFilter implements Filter, PebbleExtension {
+
+    /**
+     * Returns the {@link Filter} name.
+     * 
+     * @return the string name  - cannot be {@link null}.
+     */
+    public String name() {
+        String className = getClass().getSimpleName()
+            .replace("Filter", "")
+            .replace(PebbleFilter.class.getSimpleName(), "");
```

Comment:
If you trim "Filter" from the name you can't have "PebbleFilter" in the name afterwards :P